### PR TITLE
data conversion: qed-c to stochastic benchmark

### PR DESCRIPTION
**Pull request by PratikSathe**

### CODE REVIEW

**Comment by bernalde** [comment]
Pratik, do we need to have all the data in the repository or can we have an example file/compressed folder and have only one or two as an example? We are generating intermediate files for the analysis anyways.
Let me know what you think about this.

**Comment by PratikSathe** [comment]
I have removed the json files.

File: examples/QEDC_to_WS_conversion/conversion.py [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+# sys.path[1:1] = ["_common", "_common/qiskit", "maxcut/_common"]
+# sys.path[1:1] = ["../../_common",
+#                  "../../_common/qiskit", "../../maxcut/_common/"]
+
+# import metrics
+# import execute as ex
+
+
+
+# maxcut_style = os.path.join('..', '..', '_common', 'maxcut.mplstyle')
```
Comment (bernalde) [inline examples/QEDC_to_WS_conversion/conversion.py:27]: If these comments are not used, better remove them.

File: examples/QEDC_to_WS_conversion/conversion.py [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+        with open(pkl_file_name, "wb") as f:
+            # df = pd.read_pickle(f)
+            # gen_prop = pd.read_pickle(f)
+            pickle.dump(df, f)
+            pickle.dump(gen_prop, f)
+        
+    return df, gen_prop
+
+def do_bootstrap(df, downsample, bootstrap_iterations, metric, confidence_level = 64):
+    """_summary_
```
Comment (bernalde) [inline examples/QEDC_to_WS_conversion/conversion.py:134]: This is not quite documented. Please complete this

File: examples/QEDC_to_WS_conversion/conversion.py [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+    Args:
+        df (dataframe): indices are cobyla iterations and columns are restarts
+        downsample (int): number of elements in each sample
+        confidence_interval (int/float): default is 64
+    """
+    new_df = df.apply(lambda row: row_func(row, downsample,bootstrap_iterations, metric, confidence_level = confidence_level), axis=1, result_type="expand")
+    return new_df
+
+def pkl_to_sb_pkl_bootstrapped(pkl_folder, bs_restarts_list, bootstrap_iterations, instance_number, confidence_level, sb_target_folder = "checkpoints", sb_target_file = ""):
+    """_summary_
```
Comment (bernalde) [inline examples/QEDC_to_WS_conversion/conversion.py:145]: See comment above

File: examples/QEDC_to_WS_conversion/conversion.py [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+            new_df['instance'] = instance_number
+            
+            df_list.append(new_df)
+        
+    df = pd.concat(df_list, ignore_index=True, axis=0)
+    print("Bootstrap dataframe loaded onto memory")
+    # Store df to pkl file
+    df.to_pickle(pkl_file_name)
+    
+    return df
```
Comment (bernalde) [inline examples/QEDC_to_WS_conversion/conversion.py:209]: Add extra empty line here

**Review by bernalde — CHANGES_REQUESTED** [review]
Thanks for your PR Pratik; please address the comments.

File: examples/QEDC_to_WS_conversion/conversion.py (hunk 1 above)
Comment (PratikSathe) [reply examples/QEDC_to_WS_conversion/conversion.py]: Addressed now

File: examples/QEDC_to_WS_conversion/conversion.py (hunk 2 above)
Comment (PratikSathe) [reply examples/QEDC_to_WS_conversion/conversion.py]: Addressed now

File: examples/QEDC_to_WS_conversion/conversion.py (hunk 3 above)
Comment (PratikSathe) [reply examples/QEDC_to_WS_conversion/conversion.py]: Addressed now

File: examples/QEDC_to_WS_conversion/conversion.py (hunk 4 above)
Comment (PratikSathe) [reply examples/QEDC_to_WS_conversion/conversion.py]: Addressed now

**Comment by PratikSathe** [comment]
Comments have been addressed in a new commit.

**Review by bernalde — APPROVED** [review]

**Pull request merged**